src/FTPObject.jl

```diff
@@ -136,8 +135,18 @@ function download(
     file_name::AbstractString,
     save_path::AbstractString="";
     mode::FTP_MODE=binary_mode,
-    verbose::Union{Bool,IOStream}=false,
+    verbose=nothing,
 )
+    if verbose !== nothing
+        Base.depwarn(
+            string(
+                "The `verbose` keyword is deprecated and now needs to be supplied during ",
+                "the initialization of `FTP`.",
+            ),
+            :download,
+        )
+    end
```

Comment:
You're writing this a lot, so I'd refactor it out into a function:
```julia
function _check_verbose(f, value)
    if verbose !== nothing
        Base.depwarn("The `verbose` keyword is deprecated and now needs to be supplied " *
                     "during the initialization of `FTP`.", nameof(f))
    end
end
```
Then in each of these functions, simply write e.g. `_check_verbose(download, verbose)`.

Reply:
You're right